.github/workflows/create-release-pr.yml

```
@@ -0,0 +1,46 @@
+name: Create Release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release Version'
+        required: true
+        default: '0.0.0'
+
+env:
+  GIT_AUTHOR_NAME: mackerelbot
+  GIT_AUTHOR_EMAIL: mackerel-developers@hatena.ne.jp
+  GIT_COMMITTER_NAME: mackerelbot
+  GIT_COMMITTER_EMAIL: mackerel-developers@hatena.ne.jp
+
+jobs:
+  create-pr:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: shogo82148/actions-setup-perl@v1
+
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+
+    - uses: actions/cache@v2
+      with:
+        path: |
+          ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**\go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-${{ hashFiles('**\go.sum') }}
```

```suggestion
          ${{ runner.os }}-go-
```
Caching dependencies to speed up workflows - GitHub Docs https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows#using-the-cache-action

> You can optionally provide a list of restore-keys to use when the key doesn't match an existing cache.

See also : cache/examples.md at main · actions/cache https://github.com/actions/cache/blob/main/examples.md#go---modules